src/app/config.rs

```diff
@@ -265,6 +268,7 @@ pub const ENV_IDENTITY_DISABLED: &str = "LINKERD2_PROXY_IDENTITY_DISABLED";
 pub const ENV_IDENTITY_DIR: &str = "LINKERD2_PROXY_IDENTITY_DIR";
 pub const ENV_IDENTITY_TRUST_ANCHORS: &str = "LINKERD2_PROXY_IDENTITY_TRUST_ANCHORS";
 pub const ENV_IDENTITY_IDENTITY_LOCAL_NAME: &str = "LINKERD2_PROXY_IDENTITY_LOCAL_NAME";
+pub const ENV_IDENTITY_TAP_IDENTITY: &str = "LINKERD2_PROXY_IDENTITY_TAP_IDENTITY";
```

Comment:
This doesn't really have anything to do with proxy identity so I'd maybe put this with `ENV_TAP_DISABLED` instead of with the proxy identity vars. Also I'd consider naming it something like `LINKERD2_PROXY_TAP_NAME`.

Reply:
Good point! I'll change that